Cargo.toml

```diff
@@ -38,13 +38,14 @@ serde = "~1.0.10"
 serde_derive = "~1.0.10"
 serde_json = "~1.0.2"
 tiny-keccak = "~1.3"
-tokio-core = "~0.1.10"
+tokio-core = "=0.1.10"
```

Comment:
Hm, would be interesting to know why was this notation chosen instead of the default one?
You could put such changes in a separate commit and comment the reasons, this way I wouldn't ask in the future :)

Reply:
That was an error and gets undone in a later commit.